src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -624,6 +625,21 @@ public void onLivingUpdate()
             pickupItems();
             cleanupChatMessages();
             updateColonyServer();
+            if(worldObj.isDaytime())
+            {
+                SoundUtils.playRandomSound(worldObj, this);
+            }
+            else if(worldObj.isRaining())
+            {
+                if(isFemale())
+                {
+                    SoundUtils.playSoundAtCitizenWithChance(worldObj, this.getPosition(), FishermanSounds.Female.badWeather ,20);
+                }
+                else
+                {
+                    SoundUtils.playSoundAtCitizenWithChance(worldObj, this.getPosition(), FishermanSounds.Male.badWeather ,20);
+                }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 20 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS109)